addons/web/static/src/views/fields/field.js

```diff
@@ -324,6 +326,7 @@ Field.parseFieldNode = function (node, models, modelName, viewType, jsClass) {
                     ...archInfo,
                     limit: archInfo.limit || 40,
                     fields: models[fields[name].relation],
+                    fieldNodes: { ...relatedFields, ...archInfo.fieldNodes },
```

Comment:
I'm not confident with the way we treat relatedFields like fieldNodes. I don't know which keys they are supposed to define.